VisualPinball.Engine.PinMAME.Unity/Runtime/PinMameEnums.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Visual Pinball Engine
+// Copyright (C) 2021 freezy and VPE Team
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+namespace VisualPinball.Engine.PinMAME
+{
+	public enum PinMameMechType
+	{
+		/// <summary>
+		/// The motor turns into one direction only, and the solenoid turns it
+		/// on and off.
+		/// </summary>
+		///
+		/// <example>Sopranos pole dancers.</example>
+		OneSolenoid,
+
+		/// <summary>
+		/// Two solenoids, one for enabling the motor, one for setting the
+		/// direction.
+		/// </summary>
+		///
+		/// <example>The cannon in Terminator 2.</example>
+		OneDirectionalSolenoid,

```

Comment:
T2 does not have two solenoids. 

![4887a278e3da785a3dc60a08f9b58a564ce2ebb1](https://user-images.githubusercontent.com/1197137/139774370-7c5f07e2-7b5e-48a5-895b-abefcadbb0d7.jpg)

It works similar to this: https://www.youtube.com/embed/K7Pnxc1AncE

Reply:
Okay so it runs only into one direction, but there is a physical mechanism that makes the cannon change direction after half a spin.